.github/workflows/reusable-build-test-release.yml

```diff
@@ -45,8 +45,93 @@ on:
         required: false
 
 jobs:
+  setup-workflow:
```

Comment:
Quite a lot of logic in this job. Can you create separate action for it? It would be easier in the future to maintain it. Also maybe using Python would be easier

Reply:
Sounds Fair. Since this would need some effort and retest and we are now at the end of this Sprint and Q4 and have commitments to the dev team to deliver this feature, I have created this https://splunk.atlassian.net/browse/AQA-1044 to be worked upon in the upcoming week and make the appropriate changes accordingly.  Above comment https://github.com/splunk/addonfactory-workflow-addon-release/pull/124#discussion_r1081069795 should also be covered in the same.
CC: @bhargav-nariyani-crest